src/Validator/Constraints/PhoneNumber.php

```diff
@@ -36,7 +36,7 @@ class PhoneNumber extends Constraint
 
     public const INVALID_PHONE_NUMBER_ERROR = 'ca23f4ca-38f4-4325-9bcc-eb570a4abe7f';
 
-    protected static $errorNames = [
+    protected const ERROR_NAMES = [
```

Comment:
The class could be extended and therefore the variable could be used, right? Deleting it would be a BC break.

Can we introduce the const and add our own deprecation that we remove the variable in the next major?

Reply:
see https://github.com/odolbeau/phone-number-bundle/pull/110#discussion_r879388294

Doing the thing right requires overriding the `getErrorName` to handle both cases:
- people that extend the class and extend/use the static variable
- and people that extend the class and extends/use the const.

I've no strong opinion, but IMHO the effort to maintain the BC layer does not worth it.

Reply:
👍 works for me